<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="ATMECSINLT-060" failures="1" tests="1" name="SampleTest" time="22.727" errors="0" timestamp="24 Feb 2020 12:38:17 GMT">
  <testcase classname="com.atmecs.qa.falcondashboard.testscript.TC27_TotalTestCases" name="totalTestCases" time="22.537">
    <failure type="java.lang.AssertionError" message="expected [RUN IN - 9 secs] but found [RUN IN - 8 secs]">
      <![CDATA[java.lang.AssertionError: expected [RUN IN - 9 secs] but found [RUN IN - 8 secs]
at com.atmecs.qa.falcondashboard.helper.ValidationHelper.getDataofRow(ValidationHelper.java:29)
at com.atmecs.qa.falcondashboard.validationresults.ViewPageValidation.validateTime(ViewPageValidation.java:28)
at com.atmecs.qa.falcondashboard.testscript.TC27_TotalTestCases.totalTestCases(TC27_TotalTestCases.java:52)
at com.atmecs.qa.falcondashboard.runner.TestNGRunner.main(TestNGRunner.java:59)
at org.codehaus.mojo.exec.ExecJavaMojo$1.run(ExecJavaMojo.java:294)
at java.base/java.lang.Thread.run(Thread.java:830)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- totalTestCases -->
</testsuite> <!-- SampleTest -->
